docs/README.md

```diff
@@ -11,6 +11,8 @@ The Micro Manager manages many simulations on a micro scale and couples them to
 
 ![Micro Manager strategy schematic](images/docs-tooling-micro-manager-manager-solution.png)
 
+The Micro Manager can also compute snapshots of micro simulations given macro input parameters in an offline manner without preCICE.
```

Comment:
Lets move this below in the *What can it do?* section. You can put it after the three bullet points.